network-store-server/src/main/java/com/powsybl/network/store/server/Mappings.java

```diff
@@ -0,0 +1,773 @@
+import com.powsybl.network.store.model.CgmesSvMetadataAttributes;
+import com.powsybl.network.store.model.CimCharacteristicsAttributes;
+import com.powsybl.network.store.model.ConfiguredBusAttributes;
+import com.powsybl.network.store.model.ConnectablePositionAttributes;
+import com.powsybl.network.store.model.CoordinatedReactiveControlAttributes;
+import com.powsybl.network.store.model.DanglingLineAttributes;
+import com.powsybl.network.store.model.DanglingLineGenerationAttributes;
+import com.powsybl.network.store.model.EntsoeAreaAttributes;
+import com.powsybl.network.store.model.GeneratorAttributes;
+import com.powsybl.network.store.model.HvdcAngleDroopActivePowerControlAttributes;
+import com.powsybl.network.store.model.HvdcLineAttributes;
+import com.powsybl.network.store.model.HvdcOperatorActivePowerRangeAttributes;
+import com.powsybl.network.store.model.LccConverterStationAttributes;
+import com.powsybl.network.store.model.LimitsAttributes;
+import com.powsybl.network.store.model.LineAttributes;
+import com.powsybl.network.store.model.LoadAttributes;
+import com.powsybl.network.store.model.LoadDetailAttributes;
+import com.powsybl.network.store.model.MergedXnodeAttributes;
+import com.powsybl.network.store.model.MinMaxReactiveLimitsAttributes;
+import com.powsybl.network.store.model.NetworkAttributes;
+import com.powsybl.network.store.model.PhaseTapChangerAttributes;
+import com.powsybl.network.store.model.RatioTapChangerAttributes;
+import com.powsybl.network.store.model.ReactiveCapabilityCurveAttributes;
+import com.powsybl.network.store.model.ReactiveLimitsAttributes;
+import com.powsybl.network.store.model.RemoteReactivePowerControlAttributes;
+import com.powsybl.network.store.model.ShuntCompensatorAttributes;
+import com.powsybl.network.store.model.ShuntCompensatorLinearModelAttributes;
+import com.powsybl.network.store.model.ShuntCompensatorModelAttributes;
+import com.powsybl.network.store.model.ShuntCompensatorNonLinearModelAttributes;
+import com.powsybl.network.store.model.StaticVarCompensatorAttributes;
+import com.powsybl.network.store.model.SubstationAttributes;
+import com.powsybl.network.store.model.SwitchAttributes;
+import com.powsybl.network.store.model.TerminalRefAttributes;
+import com.powsybl.network.store.model.ThreeWindingsTransformerAttributes;
+import com.powsybl.network.store.model.ThreeWindingsTransformerPhaseAngleClockAttributes;
+import com.powsybl.network.store.model.TwoWindingsTransformerAttributes;
+import com.powsybl.network.store.model.TwoWindingsTransformerPhaseAngleClockAttributes;
+import com.powsybl.network.store.model.VoltageLevelAttributes;
+import com.powsybl.network.store.model.VoltagePerReactivePowerControlAttributes;
+import com.powsybl.network.store.model.VscConverterStationAttributes;
+import org.joda.time.DateTime;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+import java.util.UUID;
+
+public class Mappings {
```

Comment:
This class could be designed as a singleton

Reply:
Done